PocketKit/Package.swift

```diff
@@ -104,7 +104,8 @@ let package = Package(
         .target(
             name: "Analytics",
             dependencies: [
-                .product(name: "SnowplowTracker", package: "snowplow-objc-tracker")
+                .product(name: "SnowplowTracker", package: "snowplow-objc-tracker"),
+                "Sync"
```

Comment:
This is because we need `Log.` we should move utility classes to a shared package that's lightweight.

Reply:
what about `SharedPocketKit`? or are we thinking an even lighter shared package or log specific package?